model/schema/expression.yaml

```diff
@@ -12,14 +12,33 @@ imports:
 
 classes:
 
-  AnatomicalSite:
+  ExpressionExperiment:
+    mixins:
+      - AuditedObject
     description: >-
-      The developmental stage and/or age of the specimen in an annotation.
+      Defined by the gene of interest, the specimen, the assay, the reagents 
+      (Antibody, Probe), and the reference. It groups ExpressionAnnotations.
+    notes: >-
+      Specimens are defined by their AGM and/or alleles (or gel lane at MGI).
     slots:
-      - anatomical_structure
-      - anatomical_substructure
-      - cellular_component
-      - anatomical_qualifiers
+      - curie
```

Comment:
I am glad to see CURIEs/prefixed ids used universally, but I like naming fields by their roles rather than types. Might it make the schema marginally more approachable if the slot was called "unique_identifier" or simply "id"?

(it would still be typed as a CURIE)

Very minor and if the ship has sailed on this, ignore